frontend/dockerfile/docs/reference.md

```diff
@@ -894,11 +894,16 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
 # syntax=docker/dockerfile:1
 FROM ubuntu
 RUN rm -f /etc/apt/apt.conf.d/docker-clean; echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
-RUN --mount=type=cache,target=/var/cache/apt \
-  --mount=type=cache,target=/var/lib/apt \
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+  --mount=type=cache,target=/var/lib/apt,sharing=locked \
   apt update && apt-get --no-install-recommends install -y gcc
 ```
 
+Apt needs exclusive access to its data, so the caches use the option
+`sharing=locked`. This minimizes disk use and downloads but might make
```

Comment:
"... so the caches use the option `sharing=locked`, which will make sure multiple parallel builds using the same cache mount will wait for each other and not access the same cache files at the same time. You could also use
`sharing=private` if you prefer each build to create another cache directory in this case.

Reply:
I've updated the text, I hope it's good now.

Reply:
I doesn't look like you pushed your changes.

Reply:
Sorry about that, I've pushed just now.